tests/test_proxy_functional.py

```diff
@@ -158,13 +149,19 @@ async def test_secure_https_proxy_absolute_path(
     await conn.close()
 
 
-@secure_proxy_xfail(raises=AssertionError)
 @pytest.mark.parametrize("web_server_endpoint_type", ("https",))
 @pytest.mark.usefixtures("loop")
+@pytest.mark.skipif(
+    ASYNCIO_SUPPORTS_TLS_IN_TLS, reason="asyncio on this python supports TLS in TLS"
+)
```

Comment:
To ensure we don't miss any other mistakes like this, we should probably just skip based on Python version.

Reply:
f6881fd4f250cdde310fda22d175d7553f149765